src/index.js

```diff
@@ -15,7 +15,10 @@ class ShouldICliCommand extends Command {
     let question = args.question;
     let answer;
 
-    question = (question.endsWith('?')) ? question : question.concat('?');
```

Comment:
Lets add this check back in - users may potentially be use to leaving the `?` out, so I don't want this to start giving them warnings.

Reply:
Well, the feature focuses on give warnings if users forgets the '?' or if they don't include any letter. If the warnings are a problem I think there's nothing more to do.